windows/ReactTestApp/App.cpp

```diff
@@ -11,8 +11,8 @@ using namespace Windows::Foundation;
 using namespace Windows::UI::Xaml;
 using namespace Windows::UI::Xaml::Controls;
 using namespace Windows::UI::Xaml::Navigation;
-using namespace ReactTestApp;
-using namespace ReactTestApp::implementation;
+using namespace winrt::ReactTestApp;
+using namespace winrt::ReactTestApp::implementation;
```

Comment:
We didn't need these previously. Why add them now?

Reply:
`ReactTestApp` namespace is ambiguous since both `winrt::ReactTestApp` and `::ReactTestApp` are included, so need to specify which of them to use here.